source/grid/tria_description.cc

```diff
@@ -525,6 +526,380 @@ namespace TriangulationDescription
 #endif
     }
 
+    template <int dim, int spacedim>
+    struct DescriptionTemp
+    {
+      template <class Archive>
+      void
+      serialize(Archive &ar, const unsigned int /*version*/)
+      {
+        ar &coarse_cells;
+        ar &coarse_cell_vertices;
+        ar &coarse_cell_index_to_coarse_cell_id;
+        ar &cell_infos;
+      }
+
+      void
+      merge(const DescriptionTemp<dim, spacedim> &other)
+      {
+        this->cell_infos.resize(other.cell_infos.size());
```

Comment:
I'm not sure I understand this completely in terms of what cases can happen, but shouldn't you use
```suggestion
        this->cell_infos.resize(std::max(other.cell_infos.size(), this->cell_infos.size()));
```
In that case, you would also need to adjust the loop in the last part of this function to only go to `other.cell_infos.size()`.

Reply:
I have made the change. But it doe not really make a difference, since 
```cpp
description_temp_i.cell_infos.resize(tria.get_triangulation().n_global_levels());
```

;)